boot/freeldr/freeldr/include/fs/ext.h

```diff
@@ -119,8 +132,33 @@ struct ext2_block_group
   ULONG reserved[3];
 };
 
-/* The ext2 inode.  */
-struct ext2_inode
+/* The ext extent header.  */
```

Comment:
nit: if i were you i would remove the fullstop at the end of these single-line single-sentence comments.